modules/swagger-parser/src/main/java/io/swagger/parser/SwaggerResolver.java

```diff
@@ -53,4 +63,26 @@ public Swagger resolve() {
 
         return swagger;
     }
+
+    public static class Settings {
```

Comment:
It's not clear to me why we need this.  The need to add parameters to operations is dictated by the specification and the location of the parameters, not some user-specified setting.